### ISSUE TITLE
More predefined max cache size options

### ISSUE THREAD

**Issue by creeper82** [vfsfitvnm/ViMusic#1206]
### Steps to reproduce the bug

*not a bug*
At the moment song cache max size settings go from 32MB to 512MB with nothing in between. That's no problem in modern devices, but thanks to this app I gave second life to my old android phone which has barely any storage.

### Expected behavior

More options such as 128, 256MB or ability to type in custom value. It would also make sense to add 32/64MB values for song thumbnails limit

### Actual behavior

Song cache options jump from 32MB to 512MB with nothing in between

### ViMusic version

0.5.4

### Android version

Android 6

**Comment by creeper82** [comment]
After browsing the code it seems like it requires a very small modification to [that file](https://github.com/vfsfitvnm/ViMusic/blob/master/app/src/main/kotlin/it/vfsfitvnm/vimusic/enums/ExoPlayerDiskCacheMaxSize.kt)

**Comment by ghost** [comment]
Is still open ?

**Comment by vfsfitvnm** [comment]
Yes, this is a trivial thing to do. I've been deleting anything but bugs in the past months, but well-justified feature requests like this one deserve to be implemented.

Anyone can go ahead and open a pull request; but sadly, I halted the development and so the releases; you need to build this app on your own.

**Comment by NEVARLeVrai** [comment]
> Yes, this is a trivial thing to do. I've been deleting anything but bugs in the past months, but well-justified feature requests like this one deserve to be implemented.
> 
> Anyone can go ahead and open a pull request; but sadly, I halted the development and so the releases; you need to build this app on your own.

So vimusic is officially dead?
